onadata/libs/utils/api_export_tools.py

```diff
@@ -257,6 +264,8 @@ def _generate_new_export(  # noqa: C0901
         "host": request.get_host(),
         "sort": request.query_params.get("sort"),
     }
+    if export_options is not None:
+        options.update(export_options)
```

Comment:
The reason this works is because previously when creating exports synchronously, we didn't take into consideration the options passed from the requests. 

For synchronous exports, the following `fn` is used when checking if the export has been created but it's not passed to the function that actually generates the export
```python3
parse_request_export_options({})
{'include_labels': False, 'include_reviews': False, 'include_labels_only': False, 'include_hxl': True, 'remove_group_name': False, 'group_delimiter': '/', 'split_select_multiples': True, 'include_images': True, 'win_excel_utf8': False}

```
This causes a mismatch between the generated exports and when checking if the export has been generated or not

This will fix a similar issue for entity lists because the same endpoint is used